site.yml

```diff
@@ -4,6 +4,8 @@ site:
 content:
   sources:
   - url: .
+    branches:
+    - HEAD
   - url: https://github.com/owncloud/docs.git
     branches:
     - master
```

Comment:
Just replace 10.0 below with 10.1

Reply:
I've not seen that work in any testing that I've done locally, as I covered in #720, @tboerger.

Reply:
We are on the `10.0` branch, so the list below must be `master` and `10.1`, everything else is simply wrong, and that's why it's failing now.

Reply:
As I wrote in the initial PR, the reason why it is failing is that no branches were explicitly listed for the first URL. To quote [Dan Allen in the antora/users chat](https://gitter.im/antora/users?at=5c86245dd3b35423cb9d7ef2): 

remember that the default branch pattern is [master, v*]. so that first entry is effectively:

```
- url: .
  branches: [master, v*]
```

Reply:
I may have misunderstood what you were meaning, so just in case; if you mean that `master` should be the branch for the first URL and `10.1` should replace master for the second, please confirm.

Reply:
It's already applied what I was talking about.

```diff
-   - '10.0'
+   - '10.1'
```